libavpipe/src/avpipe_xc.c

```diff
@@ -1275,22 +1296,38 @@ prepare_audio_encoder(
         encoder_context->codec_context[index]->channel_layout = AV_CH_LAYOUT_STEREO;    // AV_CH_LAYOUT_STEREO is av_get_default_channel_layout(encoder_context->codec_context[index]->channels)
     }
 
-    elv_dbg("ENCODER channels=%d, channel_layout=%d (%s), sample_fmt=%s",
+    elv_dbg("ENCODER channels=%d, channel_layout=%d (%s), sample_fmt=%s, sample_rate=%d",
```

Comment:
Do we need a debug log after the sample_rate logic below?  Or is that recorded somewhere else?
(so we can record both the parameter and the actual sample rate we will end up using)

Reply:
We can figure this out by comparing input and output sample rate.